src/main/java/com/adyen/service/balancePlatform/AccountHolders.java

```diff
@@ -0,0 +1,53 @@
+package com.adyen.service.balancePlatform;
+
+import com.adyen.Client;
+import com.adyen.Service;
+import com.adyen.constants.ApiConstants;
+import com.adyen.model.balanceplatform.AccountHolder;
+import com.adyen.model.balanceplatform.AccountHolderInfo;
+import com.adyen.model.balanceplatform.JSON;
+import com.adyen.model.balanceplatform.PaginatedBalanceAccountsResponse;
+import com.adyen.service.exception.ApiException;
+import com.adyen.service.resource.balancePlatform.BalancePlatformResource;
+
+import java.io.IOException;
+import java.util.Map;
+
+
+public class AccountHolders extends Service {
+
+    public AccountHolders(Client client) {
+        super(client);
+        new JSON();
+    }
+
+    public AccountHolder create(AccountHolderInfo request) throws IOException, ApiException {
+        String jsonRequest = request.toJson();
+        BalancePlatformResource resource = new BalancePlatformResource(this, "/accountHolders");
+        String jsonResult = resource.request(jsonRequest);
+        return AccountHolder.fromJson(jsonResult);
+
+    }
+
+    public AccountHolder retrieve(String accountHolderId) throws IOException, ApiException {
+        BalancePlatformResource resource = new BalancePlatformResource(this, String.format("/accountHolders/%s", accountHolderId));
+        String jsonResult = resource.request(null, ApiConstants.HttpMethod.GET);
+        return AccountHolder.fromJson(jsonResult);
+    }
+
+    public AccountHolder update(String accountHolderId, AccountHolder request) throws IOException, ApiException {
+        String jsonRequest = request.toJson();
+        BalancePlatformResource resource = new BalancePlatformResource(this, String.format("/accountHolders/%s", accountHolderId));
+        String jsonResult = resource.request(jsonRequest, ApiConstants.HttpMethod.PATCH);
+        return AccountHolder.fromJson(jsonResult);
+    }
+
+    public PaginatedBalanceAccountsResponse list(String accountHolderId, Map<String, String> params) throws IOException, ApiException {
+        BalancePlatformResource resource = new BalancePlatformResource(this, String.format("/accountHolders/%s/balanceAccounts", accountHolderId));
+        String jsonResult = resource.request(null, null, ApiConstants.HttpMethod.GET, params);
```

Comment:
The `params` argument is used for "path parameters" (e.g. `{id}`) and not for query string params (e.g. `?limit=5&offset=10`). I also working on this in another ticket, I'm planning to add it to `com.adyen.model.RequestOptions`.